### ISSUE TITLE
Nested accordions

### ISSUE THREAD

**Issue by viktorvuka** [michu2k/Accordion#16]
Is there a way to nest an accordion into another one with v3 ?

Such as : 

Primary Accordion
- Choice 1
-- Content
- Choice 2
-- SubChoice 1
--- Content
-- SubChoice 2
--- Content

**Comment by michu2k** [comment]
Hi @viktorvuka 
It is possible, but a bit complicated. You can define a second accordion with your `subChoices` but you have to change `elementClass`, `triggerClass` and `panelClass` for that accordion.

example:
```javascript
new Accordion('.accordion-container2', {
    elementClass: 'ac2',
    triggerClass: 'ac2-trigger',
    panelClass: 'ac2-panel',
});
```

I am currently working on implementing code for a nested accordion so it will be much simpler in the next update :)

**Comment by michu2k** [comment]
@viktorvuka 
Nested accordions landed on the `prelease` branch. You can download the files and check that everything is working properly. You don't need any addiontal configuration, just define another accordion.

**Comment by viktorvuka** [comment]
@michu2k Thank you ! I will definitely try that soon !

**Comment by michu2k** [comment]
Available on master (v3.1)
